public/app/features/live/data/StreamingDataFrame.ts

```diff
@@ -1,20 +1,8 @@
-import { DataFrame, Field, FieldDTO, FieldType, Labels, QueryResultMeta } from '../types';
-import { ArrayVector } from '../vector';
-import { DataFrameJSON, decodeFieldValueEntities, FieldSchema } from './DataFrameJSON';
-import { guessFieldTypeFromValue, toFilteredDataFrameDTO } from './processDataFrame';
-import { join } from '../transformations/transformers/joinDataFrames';
+import { DataFrame, Field, FieldDTO, FieldType, Labels, QueryResultMeta, DataFrameJSON, decodeFieldValueEntities, FieldSchema, guessFieldTypeFromValue, ArrayVector, toFilteredDataFrameDTO } from '@grafana/data';
+import { join } from '@grafana/data/src/transformations/transformers/joinDataFrames';
```

Comment:
nit: Same thing here. It should be imported from the external API of `@grafana/data`. If this file e.g. moves location it will break this import even if it still is exported from `@grafana/data`

Reply:
fixed this one